sympy/geometry/ellipse.py

```diff
@@ -1022,6 +1022,32 @@ def auxiliary_circle(self):
         """
         return Circle(self.center, Max(self.hradius, self.vradius))
 
+    def director_circle(self):
+        """Returns a Circle consisting of all points where two perpendicular tangent lines to the ellipse cross each other.
```

Comment:
This line is too long for [PEP-8](https://www.python.org/dev/peps/pep-0008/#maximum-line-length)

> Limit all lines to a maximum of 79 characters.

> For flowing long blocks of text with fewer structural restrictions (docstrings or comments), the line length should be limited to 72 characters.

Though it strictly says 72-character for docstrings. I think you can split it to fit in 80-terminal window.